### DISCUSSION TITLE
please show a woocommerce development

### DISCUSSION THREAD

**Discussion by vahidalvandi** [tailpress/tailpress#187] [Q&A]
please show a woocommerce development

**Comment by jarednthomas** [comment]
I've made a Woocommerce compatible theme out of Tailpress by following the file and folder structure outlined in the [Woocommerce Documentation ](https://woocommerce.com/document/template-structure/) and adding theme supports:

add_theme_support( 'woocommerce' );
add_theme_support( 'wc-product-gallery-lightbox' );
add_theme_support( 'wc-product-gallery-zoom' );
